sensor_stack/sparton/src/sparton.py

```diff
@@ -241,10 +246,71 @@ def getData():
     allData.data[12] = temp
     
     return allData
-    
- 
-if __name__ == '__main__':
-    """    
+
+def getOrientation():
+
+    global roll
+    global pitch
+    global yaw
+
+    quaternion = 
+
+    return quaternion
+
+def getOrientationCovariance():
+
+    # global roll
+    # global pitch
+    # global yaw
+
+    cov_mat = [0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
+
+    return cov_mat
+
+def getAngularVelocityCovariance():
+
+    # global gx
+    # global gy
+    # global gz
+
+    cov_mat = [0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
+
+    return cov_mat
+
+
+def getLinearAccelerationCovariance():
+
+    # global ax
+    # global ay
+    # global az
+
+    cov_mat = [0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
+
+    return cov_mat
+
+def getQuaternion():
```

Comment:
I actually knew that `tf` contained such a transformation function, but the documentation was simply just horrible - I knew there's a function & still couldn't find it! :disappointed: 

The guy who posted the answer also shares the same point of view as me. Blame the documentation.  :-P